src/base/bittorrent/session.cpp

```diff
@@ -1124,6 +1124,7 @@ void Session::configure(libtorrent::settings_pack &settingsPack)
             }
         }
 
+        settingsPack.set_str(libt::settings_pack::outgoing_interfaces, networkInterface().toStdString());
```

Comment:
would it be better to move this to under line 1110 & line 1122 and become
`settingsPack.set_str(libt::settings_pack::outgoing_interfaces, interfacesStr);`?